openshift/4.0/templates/database/mssql-build.yaml

```diff
@@ -75,20 +80,23 @@ objects:
     apiVersion: v1
     metadata:
       name: ${APP_NAME}-${ROLE_NAME}
+      namespace: ${PROJECT_NAMESPACE}-${ENV_NAME}
       annotations:
         description: Keeps track of changes in the database image
       labels:
         app: ${APP_NAME}
         role: ${ROLE_NAME}
+
   # The build config that will be created will be named for the branch you created it for.
   - kind: BuildConfig
     apiVersion: v1
     metadata:
-      name: ${APP_NAME}-${ROLE_NAME}-${GIT_REF}
+      name: ${APP_NAME}-${ROLE_NAME}.${GIT_REF}
```

Comment:
Changed syntax convention for the branch from `-dev` to `.dev` and `-master` to `.master`.  Due to `dev` being easily confused with the namespaced environments rather than the branch name.  Just a minor idea to make the user think differently.  I would have prefered `#dev` or `:dev`, but those characters are not allowed.